plugins/pulp_python/plugins/utils.py

```diff
@@ -0,0 +1,5 @@
+import re
+
+
+def normalize_project_name(name):
+    return re.sub(r"[-_.]+", "-", name).lower()
```

Comment:
I would prefer we just used the same function as with Pulp 3, unless there is a reason otherwise.

https://github.com/pulp/pulp_python/blob/3.0-dev/pulp_python/app/tasks/publish.py#L145